free-2.md

```diff
@@ -11,41 +11,106 @@ _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+This idea meets the need of people who want to figure out what song they 
+heard at the mall or on T.V. but don't remember much of it. Maybe they vaguely
+remember some lyrics but not so much the tune. Or maybe they only remember the 
+rhythm but don't remember any of the specific lyrics. Or maybe they even just 
+remember how they felt while listening to the song. Right now, it's really 
+difficult to figure out a song when you only have a vague recollection of it.
+Sure, there are song identification apps like Shazam, but it requires you to have
+your phone and internet access at that very moment the song is playing. But you
+might only realize how much you like the song right after it's over. With my DSL,
+the hope is that people will be able to identify the songs they heard in the past,
+even if they don't remember much of it, and allow them to discover new ones along 
+the way!
+
 
 ### Why a language?
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
+A DSL would give the user a large range of methods for identifying a song.
+There are basic things like lyrics that the user could input, but they could
+input many other features such as the song's genre, mood, volume, and even rhythm
+patterns. Rather than acting like a simple search engine, this DSL would allow the 
+user to use many different kinds of parameters and combine this information to 
```

Comment:
This seems really hard. You're going to have to do a lot of systems work on things like signal processing (and also extracting fragments that may be inaccurate) that it makes me think it won't be worthwhile.